common/log/tag/tags.go

```diff
@@ -58,6 +59,11 @@ func Timestamp(timestamp time.Time) Tag {
 	return newTimeTag("timestamp", timestamp)
 }
 
+// Timestamp returns tag for Timestamp
+func TimestampPtr(t *time.Time) Tag {
```

Comment:
```suggestion
func TimePtr(t *time.Time) Tag {
```

Reply:
Seems func naming is based off tag string key, in this case - `"timestamp"`